packages/selection/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@zendeskgarden/container-selection",
+  "description": "Containers relating to selection in the Garden Design System",
+  "license": "Apache-2.0",
+  "author": "Zendesk Garden <garden@zendesk.com>",
+  "homepage": "https://garden.zendesk.com/",
+  "repository": "https://github.com/zendeskgarden/react-containers",
+  "bugs": {
+    "url": "https://github.com/zendeskgarden/react-containers/issues"
+  },
+  "version": "0.0.0",
+  "main": "./dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "../../utils/scripts/build.sh"
+  },
+  "peerDependencies": {
+    "prop-types": "^15.6.1",
+    "react": "16.8.2",
```

Comment:
Both `react` and `react-dom` should be "open" `^`, correct? And I think we may be able to pin it to the v16.8 release with hooks.

Reply:
correct, will change.